test.js

```diff
@@ -153,17 +153,18 @@ describe('when connection is established', function () {
 			resp.resume();
 			var req2 = http.get(reqOpts, function (resp) {
 				resp.resume();
-				keepAliveAgent.destroy();
+				socket.destroy();
```

Comment:
Should I remove 0.10 and 0.12 from travis [config](https://github.com/floatdrop/timed-out/blob/41347e64b682558f5d6fcd4afa036757eedb7c00/.travis.yml#L3-L4)?